sdk/ai/Azure.AI.Inference/tests/InferenceClientTestEnvironment.cs

```diff
@@ -21,6 +21,8 @@ public class InferenceClientTestEnvironment : TestEnvironment
         public string TestImagePngInputPath => GetRecordedVariable("AZUREAI_TEST_IMAGE_PNG_INPUT_PATH");
         public string AoaiAudioEndpoint => GetRecordedVariable("AOAI_AUDIO_ENDPOINT");
         public string AoaiAudioKey => GetRecordedVariable("AOAI_AUDIO_KEY", options => options.IsSecret());
+        public string PhiAudioEndpoint => GetRecordedVariable("PHI_AUDIO_ENDPOINT");
+        public string PhiAudioKey => GetRecordedVariable("Phi_AUDIO_KEY", options => options.IsSecret());
```

Comment:
PHI_ vs Phi_

Reply:
Interesting that it still worked... guess it's not case sensitive